modules/ps_newproducts/views/templates/hook/ps_newproducts.tpl

```diff
@@ -23,13 +23,14 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  *}
 
-<section class="featured-products mt-3">
-  <h2 class="h2 products-section-title text-uppercase">
-    {l s='New products' d='Shop.Theme.Catalog'}
-  </h2>
-  {include file="catalog/_partials/productlist.tpl" products=$products productClass="col-6 col-lg-4 col-xl-3"}
-  <a class="all-product-link float-start float-md-end h4" href="{$allNewProductsLink}">
-    {l s='All new products' d='Shop.Theme.Catalog'}<i class="material-icons">&#xE315;</i>
-  </a>
-</section>
-
+<section class="new-products mt-3">
+    <div class="container">
+        {include file="components/section-title.tpl" title={l s="New products" d="Shop.Theme.Catalog"}}
+        {include file="catalog/_partials/productlist.tpl" products=$products cssClass="row" productClass="col-6 col-lg-4 col-xl-3"}
+        <div class="new-products-footer text-center">
+            <a class="all-product-link btn btn-outline-primary" href="{$allNewProductsLink}">
+                {l s='All new products' d='Shop.Theme.Catalog'}<i class="material-icons">&#xE315;</i>
+            </a>
+        </div>
+    </div>
+</section>
```

Comment:
Missing empty line

You can configure your IDE so it adds empty lines automatically at the end of the file :)

Reply:
editorconfig maybe?